temperature_regulator/server.py

```diff
@@ -0,0 +1,21 @@
+import socket
+
+if __name__ == "__main__":
```

Comment:
Taka ogólna uwaga to może warto byłoby stosować się trochę do sztuki programistycznej i zrobić jakieś klasy? Bo jak na razie to mamy dwa mainy i wszystko jest w tych mainach (co mam nadzieje ulegnie zmianie w przyszłości). Proponowałbym od samego poczatku zrobić dwie klasy Server i Client i poumieszczać tam jakieś metody które moglibyśmy użyć w odpowiednich mainach